packages/pie-toolbox/src/code/render-ui/color.js

```diff
@@ -20,6 +20,7 @@ export const defaults = {
   SECONDARY_DARK: pink[900],
   BACKGROUND: 'rgba(255,255,255,0)',
   SECONDARY_BACKGROUND: 'rgba(241,241,241,1)',
+  BLACK: '#000000',
```

Comment:
le't remove `BLACK: '#000000'` from here and you can use `theme.palette.common.black` where you want to use black color

Reply:
Sure Andrei, I will remove and update the PR.